Content.Server/Weapon/Melee/MeleeWeaponSystem.cs

```diff
@@ -88,7 +90,7 @@ private void OnClickAttack(EntityUid uid, MeleeWeaponComponent comp, ClickAttack
 
                     if (target.TryGetComponent(out IDamageableComponent? damageableComponent))
                     {
-                        damageableComponent.ChangeDamage(comp.DamageType, comp.Damage, false, owner);
+                        damageableComponent.ChangeDamage(comp.DamageType != null ? damageableComponent.GetDamageType(comp.DamageType) : damageableComponent.GetDamageType("Blunt") , comp.Damage, false, owner);
```

Comment:
If blunt is intended to be the default then this should be the default of the field for the melee weapon component damage type field, otherwise this should not deal any damage when its null.